eel/src/storage_persister.rs

```diff
@@ -447,7 +449,6 @@ impl<ChannelSigner: Sign> Persist<ChannelSigner> for StoragePersister {
 
 fn sync_persist_monitor_remotely(
```

Comment:
Yep, you're right; we should have a limit. I removed the previous one based on `ChainMonitor::list_pending_monitor_updates()` because it was causing problems. I think we can't call it before `persist_new_channel()` returns, probably because of a lock. I've added a limit on the max number of retries in [b528e15](https://github.com/getlipa/lipa-lightning-lib/pull/204/commits/b528e1561e92674e549132fb9bfa8df362266991).